assets/models/foundrylocal/mistralai-mistral-7b-instruct-v0-2-trtrtx-gpu/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: foundrylocal/olive-trt-models/mistralai-Mistral-7B-Instruct-v0-2
```

Comment:
In container, it's 'v0.2', not 'v0-2'. You can double check from the storage account.